src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/config/HmppsBookAVideoLinkApiExceptionHandler.kt

```diff
@@ -71,10 +72,10 @@ class HmppsBookAVideoLinkApiExceptionHandler : ResponseEntityExceptionHandler()
   fun handleCaseLoadAccessException(e: CaseloadAccessException): ResponseEntity<ErrorResponse> {
     log.info("Case load access exception: {}", e.message)
     return ResponseEntity
-      .status(NOT_FOUND)
+      .status(FORBIDDEN)
```

Comment:
404 NOT_FOUND only works for GET scenario so changed to 403 FORBIDDEN to cater for all other scenarios.